README.md

```diff
@@ -433,6 +439,10 @@ It also has any other field from the [`basic rule`](#basic-rule) (with the excep
 	- **Optional**: Defaults to `false`.
 - **minRank**: Must be a number.
 	- **Required**
+
+##### Note
+The fellows rule will never request reviewers, even if `request-reviewers` rule is enabled.
```

Comment:
I feel like adding a reason for this would be good.
I understand it's because it would result in too many reviewers requested for lower levels of fellows.
I think it should be put down in writing for future reference.